nrt/data/DisconnectConnectLoad/DisconnectConnectLoad.dyd

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+    Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```

Comment:
2021 (same for all files)

Reply:
Maybe it should go into example?

Reply:
2021 done

Reply:
for example I don't have an opinion

Reply:
Usually this kind of test goes in dynawo-rte/nrt/data/standaloneCases

Reply:
sounds good to me